test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -693,6 +694,9 @@ public static void assertVersionSerializable(Version version, Streamable streama
             // and the readFrom method throws an exception if called
             Streamable newInstanceFromStream = tryCreateFromStream(streamable, input);
             if (newInstanceFromStream == null) {
+                if (newInstance instanceof ClusterAllocationExplainRequest) {
+                    System.out.println("what?");
```

Comment:
This looks like a leftover.

Reply:
oh me oh my. indeed. I was debugging. will remove. thanks!

Reply:
If you add `// nocommit` any time that you add a debug line like this, the build will help you remember to remove it (a pre-commit check will fail).

Reply:
thanks for the protip! I will